### PR TITLE
Feat/thumbs up down and tags

### PR DESCRIPTION
**Pull request by aeschi**
[https://app.asana.com/0/1205231177119914/1206888585206828](https://app.asana.com/0/1205231177119914/1206888585206828) 

feedback still needs to be saved with the question and result data set

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -186,7 +196,13 @@ function App() {
 					abortController.current?.signal,
 				);
 				if (userRequest) historyEntry = userRequest;
-				setResultHistory((prev) => [userRequest, ...prev]);
+
+				console.log("resultHistory", resultHistory);
```
Comment (Jaszkowic) [inline src/app/page.tsx:200]: console.log() can be removed

**Review by Jaszkowic — CHANGES_REQUESTED** [review]
Found one edge case:

- This PR migrates the history from a custom implementation to Zustand
- On first page load, it recovers the old history and puts it into the new history (works fine)
```
	// For backwards compatibility with old localStorage history, we initially load the parly-history
		// save it in the new Zustand store and remove it from localStorage
		if (localStorage.getItem("parla-history")) {
			setResultHistory(JSON.parse(localStorage.getItem("parla-history")!));
			localStorage.removeItem("parla-history");
		}
```
- Now we have old entries in there with outdated (long) IDs 
- When clicking on a history entry, the request is loaded (works fine) from localstorage
- Since it is a old request which was not yet saved to database, it is not possible to add feedback
- However, the UI still shows the thumbs up / down buttons. On click on those buttons, there is an error message.

Simple fix: Don't show the feedback option for old requests.

**Comment by raphael-arce** [comment]
@Jaszkowic I have added a guard close to prevent feedback on older results. Could you have another look? thx!

**Review by Jaszkowic — APPROVED** [review]
LGTM

**Pull request merged**